doc/final_report.md

```diff
@@ -0,0 +1,90 @@
+
+As this population is so vulnerable, the other key factor that should be investigated is if there is a difference in the length of the sentence between the indigenous and non-indigenous groups. This is an important question to infer as it can lead to further studies on the type of charges either group is subject too and whether particular laws disadvantage the indigenous group if a significant difference is found. Additionally, the study could also lead to an exploration of biases present in the current legal system when sentencing. 
+
+# Methods 
+
+## Data 
+
+The data set used for this study is the Offender Profile from 2017-2018 released by the Correctional Service of Canada. The link to this site can be found [here](https://open.canada.ca/data/en/dataset/844ff1e3-e137-41be-9ebe-6bd9843c1a53). Each entry in the dataset corresponds to a single offender serving a two or more year long sentence. The demographic details such as age, gender and marital status at year end are provided for each entry. This was retrieved from the Offender Management System (OMS).
+
+##  Analysis
+
+A hypothesis test with a significance level of 0.05 was carried out to determine if there was a statistically significant difference in the medians of the sentence length between the indigenous and non-indigenous groups. The features used from the data set to complete this analysis were the racial grouping, which was already separated into indigenous and non-indigenous, and aggregate sentence length. The median was selected as the measure of central tendency and the permutation test under the null hypothesis. The null hypothesis stated that there was no difference in the population medians in the sentence length between indigenous and non-indigenous inmates. The alternate hypothesis stated that there was a difference in the population median between indigenous and non-indigenous inmates.
+
+The R and Python programming languages [@R; @Python] and the following R and Python packages were used to perform the analysis:  docopt [@docopt],  knitr [@knitr], tidyverse [@tidyverse], infer[@infer], docopt [@docoptpython]. The code used to perform the analysis and create this report can be found here: https://github.com/UBC-MDS/DSCI_522_inference_on_indigenous_vs_non_indigenous_sentence_length_differences.
+
+
+# Results and discussion 
+
+Firstly, the only relevant features for our hypothesis test were the ‘aggregate sentence length’ and the ‘racial grouping column’. We decided to use the ‘racial grouping’ column over the race column as it already contained two groups as required by the hypothesis testing: ‘indigenous’ and ‘non-indigenous’. All other columns were dropped. The count of observations in either group was `r COUNT_IND` and `r COUNT_NON_IND`, meaning we had a large enough sample size to carry out a t-test if need be. 
```

Comment:
The only relevant columns for our hypothesis test were the ‘aggregate sentence length’ and the ‘racial grouping’. ‘Racial grouping’ column was included in the report and summarized data from the ‘race’ columns into two categories.  “Aboriginal” corresponded to the offenders with a race of “First Nations”, “Métis”, or “Inuit”.  “Non Aboriginal” corresponded to all others. Note that this information was self-reported.

All other columns were dropped. The count of observations in either group was `r COUNT_IND` and `r COUNT_NON_IND`, meaning we had a large enough sample size to carry out a t-test.